### DISCUSSION TITLE
http request for switching charger on and off

### DISCUSSION THREAD

**Discussion by angryc** [PatrikTrestik/homeassistant-solax-http#38] [General]
Hi @PatrikTrestik,
thanks for your great work!
Could you please give me a hint how to switch the charger on and off with a http request?
I am already able to read the data from it. And I want to use the data from the charger and the inverter in my own script to start/stop charging.
Thank you!

**Comment by angryc** [comment]
I tried the Cloud API, but I get "No Auth!", so maybe a normal user is not allowed to control the charger via cloud.

**Reply by PatrikTrestik** [reply]
Hi @angryc 

This is discussion related to Home Assistant integration, not API itself.
Unfortunately there is no documentation about local API from SolaX. All what you find in code is investigation and trial and error research.

You can find some documentation about Cloud API on SolaX cloud pages.

**Comment by PatrikTrestik** [comment]
Correct entity for changing EVC mode is "Charger Use Mode".
